app/controllers/concerns/search_aggregator.rb

```diff
@@ -59,22 +67,29 @@ def search_concept_label(label)
         pref_lab.downcase.include?(@search_query.downcase) || @search_query.downcase.include?(pref_lab.downcase)
       end.first || label.first
     end
-    
+
     label
   end
   def search_result_elem(class_object, ontology_acronym, title)
```

Comment:
refactor this function, `is_federation_external_class(class_object) ?` is repeated too much.